scripts/export-app.sh

```diff
@@ -0,0 +1,48 @@
+# Check if correct number of arguments are provided
+if [ "$#" -ne 7 ]; then
+    echo "Usage: $0 <app_id> <splunk_stack_username> <splunk_stack_password> <stack_name> <environment> <acs> <output_file_name>"
+    exit 1 
+fi
+
+# Set variables from command-line arguments
+APP_ID="$1"
+SPLUNK_STACK_USERNAME="$2"
+SPLUNK_STACK_PASSWORD="$3"
+STACK_NAME="$4"
+ENVIRONMENT="$5"
+ACS="$6"
+OUTPUT_FILE_NAME="$7"
+
+# Generate Splunk Cloud (stack) token
+echo "Generating Splunk Cloud token..."
+echo "https://$ENVIRONMENT/$STACK_NAME/$ACS/tokens"
+STACK_RESPONSE=$(curl -u $SPLUNK_STACK_USERNAME:$SPLUNK_STACK_PASSWORD \
+        -X POST https://$ENVIRONMENT/$STACK_NAME/$ACS/tokens \
+        --header 'Content-Type: application/json' \
+        --data-raw '{"user" : "'$SPLUNK_STACK_USERNAME'",
+                    "for sc_admins" : "install app victoria",
+                    "expiresOn" : "+5d",
+                    "audience" : "acs-test"}' )
```

Comment:
Love it!
Looks like we may be able to target ES vs Adhoc,
 so add another variable for ENVIRONMENT that has one of the following defined.
adhc=
es=
itsi=

Reply:
@macycron ACS=$5 is just assigning the ACS variable to the 6th parameter of the command.  One question, we already have an ENVIRONMENT variable that corresponds to the host, would this one you are suggesting work similiar? Or where in the code would these values go? Looking at the docs I can't find where we can specify Adhoc vs ES